### PR TITLE
Devcontainer updates

### PR DESCRIPTION
**Pull request by Will-Cooper**
wget -O needs the filename

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	"forwardPorts": [8000],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "pip install -r requirements.txt && wget -O https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db"
+	"postCreateCommand": "pip install -r requirements.txt && wget -O SIMPLE.db https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db"
```
Comment (Will-Cooper) [inline .devcontainer/devcontainer.json:14]:
```suggestion
	"postCreateCommand": "pip install -r requirements.txt && wget -O SIMPLE.db https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db && python simple_app/app_simple.py"
```

**Review by Will-Cooper — COMMENTED** [review]
Can launch website as well

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	"forwardPorts": [8000],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "pip install -r requirements.txt && wget -O https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db"
+	"postCreateCommand": "pip install -r requirements.txt && wget -O SIMPLE.db https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db && python simple_app/app_simple.py"
```
Comment (dr-rodriguez) [inline .devcontainer/devcontainer.json:14]: Can we add the `--debug` option you had built for running the app? That way any edits should automatically be reflected without having to restart it

Comment (Will-Cooper) [reply .devcontainer/devcontainer.json:14]:
```suggestion
	"postCreateCommand": "pip install -r requirements.txt && wget -O SIMPLE.db https://raw.githubusercontent.com/SIMPLE-AstroDB/SIMPLE-binary/main/SIMPLE.db && python simple_app/app_simple.py -d"
```

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**